src/backend/marsha/core/tests/test_views_lti_cache.py

```diff
@@ -22,6 +22,10 @@
 class CacheLTIViewTestCase(TestCase):
     """Test caching in LTI views."""
 
+    def setUp(self):
+        # warmup request
+        self.client.get("/")
```

Comment:
If you put it here it will played before each test which is not what we want? Maybe just put it in the test where we check performance?

Reply:
`self.assertTrue(elapsed < 0.1)` is everywhere…

Reply:
And it didn't work :(
https://app.circleci.com/pipelines/github/openfun/marsha/2339/workflows/82aadcff-45bc-4065-bbbf-be4e04ccc372/jobs/88570

Reply:
:hand_over_mouth:

Reply:
It's more a Video.objects.get() that you need. The '/' url has nothing behind :man_shrugging:

Reply:
Ah ok, "/" get request worked on my laptop.
I'll try a database request.

Reply:
I tried several db requests without success

Reply:
The warmup request if so that this test passes when it is played alone. It has no impact on the CI that runs all the tests.

To fix the CI:
- short term : increase the limit and open an issue
- long term : fix waffle to optimize requests